app/Services/SimBriefService.php

```diff
@@ -39,10 +41,18 @@ public function downloadOfp(
         string $ofp_id,
         string $flight_id,
         string $ac_id,
-        array $fares = []
+        array $fares = [],
+        string $sb_userid = null,
+        string $sb_static_id = null
     ) {
         $uri = str_replace('{id}', $ofp_id, config('phpvms.simbrief_url'));
 
+        if ($sb_userid && $sb_static_id) {
+            // $uri = str_replace('{sb_userid}', $sb_userid, config('phpvms.simbrief_update_url'));
+            // $uri = str_replace('{sb_static_id}', $sb_static_id, $uri);
+            $uri = 'https://www.simbrief.com/api/xml.fetcher.php?userid='.$sb_userid.'&static_id='.$sb_static_id;
```

Comment:
Use the URL that's on the config file; you've got the commented out code up there

Reply:
That was the problem i faced, when I use the config (commented out lines) it generates an error which I was not able to solve. And this was the reason I wrote a note about this line in the main issue text, kindly requested you to check that part.

Reply:
`[2021-06-06 13:18:48] prod.ERROR: simplexml_load_string(): Entity: line 1: parser error : Start tag expected, '&lt;' not found
{"userId":3,"exception":"[object] (ErrorException(code: 0): simplexml_load_string(): Entity: line 1: parser error : Start tag expected,
'&lt;' not found at /app/Services/SimBriefService.php:73)`

This is the error when switched to config for reading the url

Reply:
Did you output the URL after the str_replace to make sure the replace was completed properly?